t/randomize-classes.t

```diff
@@ -0,0 +1,14 @@
+use strict;
+use warnings;
+
+#use Test::More;
+use Test::More skip_all => 'cannot guarantee that randomized class list is not ABCDE';
+
+use Test::Class::Moose::Load 't/randomlib';
+use Test::Class::Moose::Runner;
+
+my $tcmr = Test::Class::Moose::Runner->new( randomize_classes => 1 );
+
+isnt( join('',$tcmr->test_classes), 'ABCDE', 'hopefully got a random list of classes' );
```

Comment:
Maybe make this an author test and/or mark it as TODO. Adding a test that's always skipped isn't very useful.